dockstore-webservice/src/main/resources/import.sql

```diff
@@ -31,5 +31,5 @@ CREATE UNIQUE INDEX organization_name_index on organization (LOWER(name));
 CREATE UNIQUE INDEX collection_name_index on collection (LOWER(name), organizationid);
 -- JPA doesn't seem to understand deferrable constraints - this is a terrible fix to alter the auto generated constraints
 -- we also can't seem to customize the naming ... sigh
-ALTER TABLE tag alter CONSTRAINT fk_6yv7s5k85a0emvco4ikb6t5ps DEFERRABLE INITIALLY DEFERRED;
-ALTER TABLE workflowversion alter CONSTRAINT fk_25o6afine0dolbhs0hqhnh4ao DEFERRABLE INITIALLY DEFERRED;
\ No newline at end of file
+ALTER TABLE tag ADD CONSTRAINT fk_tagVersionMetadata FOREIGN KEY(id) REFERENCES public.version_metadata (id) DEFERRABLE INITIALLY DEFERRED;
```

Comment:
Would you mind explaining what happened here? And does this mean you can alter the comment above it?

Reply:
Sure, so this file is invoked when you run hbm2ddl (i.e. dropwizard with hibernate set to create mode). The idea is that our script ( https://github.com/dockstore/dockstore/blob/develop/propose_migration.sh ) compares what JPA thinks the database should be with what our migrations.xml files propose. 

The problem is that JPA can't create many of the postgres-specific structures that we use since JPA is standard across all databases.

In this case, I earlier in this PR I had the JPA create an incomplete constraint which we modified ... but I was unable to specify the name so I altered the constraint. In the end, I created it completely outside of JPA so I was able to specify the name (and deleted the comment above)